packages/nouns-subgraph/src/stream-escrow.ts

```diff
@@ -0,0 +1,117 @@
+import { ETHStreamed, Stream, StreamFastforward } from './types/schema';
+import {
+  AllowedToCreateStreamChanged,
+  DAOExecutorAddressSet,
+  ETHRecipientSet,
+  ETHStreamedToDAO,
+  NounsRecipientSet,
+  StreamCanceled,
+  StreamCreated,
+  StreamFastForwarded,
+  StreamsForwarded,
+} from './types/StreamEscrow/StreamEscrow';
+import { BIGINT_ZERO } from './utils/constants';
+import {
+  genericUniqueId,
+  getCurrentStream,
+  getOrCreateStreamCreationPermission,
+  getOrCreateStreamsOfNoun,
+  getStreamEscrowState,
+} from './utils/helpers';
+
+export function handleETHStreamedToDAO(event: ETHStreamedToDAO): void {
+  const es = new ETHStreamed(genericUniqueId(event));
+  es.amount = event.params.amount;
+  es.createdTimestamp = event.block.timestamp;
+  es.createdBlock = event.block.number;
+  es.save();
+
+  const s = getStreamEscrowState();
+  s.totalAmountStreamedToDAO = s.totalAmountStreamedToDAO.plus(event.params.amount);
+  s.save();
+}
+
+export function handleStreamCreated(event: StreamCreated): void {
+  const nounId = event.params.nounId.toString();
+  const streamId = genericUniqueId(event);
+
+  const s = new Stream(streamId);
+  s.createdTimestamp = event.block.timestamp;
+  s.createdBlock = event.block.number;
+  s.noun = nounId;
+  s.totalAmount = event.params.totalAmount;
+  s.streamLengthInTicks = event.params.streamLengthInTicks;
+  s.ethPerTick = event.params.ethPerTick;
+  s.lastTick = event.params.lastTick;
+  s.canceled = false;
+  s.cancellationRefundAmount = BIGINT_ZERO;
+  s.save();
+
+  const son = getOrCreateStreamsOfNoun(nounId);
+  if (son.currentStream != null) {
+    son.pastStreams.push(son.currentStream!);
+  }
+  son.currentStream = s.id;
+  son.save();
+}
+
+export function handleStreamFastForwarded(event: StreamFastForwarded): void {
+  const nounId = event.params.nounId.toString();
+
+  const ff = new StreamFastforward(genericUniqueId(event));
+  ff.createdTimestamp = event.block.timestamp;
+  ff.createdBlock = event.block.number;
+  ff.stream = nounId;
```

Comment:
@eladmallel this looks wrong. should be `getOrCreateStreamsOfNoun(nounId).currentStream` I think?

Reply:
fixed: https://github.com/nounsDAO/nouns-monorepo/pull/869/commits/8395db0105202ca5ae175c1de8722eb8cbbef9da